torch/_dynamo/variables/builder.py

```diff
@@ -564,6 +574,10 @@ def wrap_tensor(self, value: torch.Tensor):
                     guards=self.make_guards(GuardBuilder.TENSOR_MATCH),
                     should_specialize=self.tensor_should_specialize(),
                 )
+            if graph_arg and config.fake_tensor_propagation:
+                example = tensor_variable.proxy.node.meta["example_value"]
```

Comment:
Does it matter that you're pulling out `example_value` here and not `val`?

Reply:
No, Val reads better tho